data_relay.py

```diff
@@ -41,7 +41,6 @@
         ('B', 'gpsStatus'),
         ('B', 'batteryLevel'),
         ('B', 'waypointCount'),
-	('B', 'airspeed'),
```

Comment:
The packet should be even length, I expect removing a byte without putting the padding in will break the receiving. Please test this on the actual data.